src/frontend/src/pages/AdminToolsPage/AdminToolRecruitmentTab.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Grid, Typography } from '@mui/material';
+import { NERButton } from '../../components/NERButton';
+import { Milestone } from 'shared/src/types/milestone-types';
+import { useHistoryState } from '../../hooks/misc.hooks';
+import CreateMilestoneFormModal from './RecruitmentConfig/CreateMilestoneFormModal';
+import EditMilestoneFormModal from './RecruitmentConfig/EditMilestoneFormModal';
+import MilestoneTable from './RecruitmentConfig/MilestoneTable';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { useAllMilestones } from '../../hooks/recruitment.hooks';
+
+const AdminToolsRecruitmentTab: React.FC = () => {
```

Comment:
Let's make this tab match more with the style and layout of the other tabs. Specifically it would be nice to have a matching heading, and the table should be the same style as the others in admin tools. The Finance Config tab would be a good one to emulate generally